<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisSettings</class>
 <widget class="QWidget" name="AxisSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QGroupBox" name="gridGroupBox">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Grid Line</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="5" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Line width</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="gridHorizontalCheck">
              <property name="text">
               <string>Horizontal</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="gridVerticalCheck">
              <property name="text">
               <string>Vertical</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="gridLineTypeCombo"/>
            </item>
            <item row="4" column="1">
             <widget class="GuiUtils::ColorPushButton" name="gridLineColorButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Line color</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Line type</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="gridLineWidthCombo"/>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="gridMajorCheck">
              <property name="text">
               <string>Major grids</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="gridMinorCheck">
              <property name="text">
               <string>Minor grids</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Axis Type</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="axisTitleCombo">
              <item>
               <property name="text">
                <string>Long Name</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Short Name</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Unit</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Long Name &amp; Unit</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Short Name &amp; Unit</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Thickness</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QComboBox" name="axisTickWidthCombo">
              <property name="currentText">
               <string notr="true">1</string>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="GuiUtils::ColorPushButton" name="axisTickColorButton">
              <property name="text">
               <string/>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Tick Label</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Prefix</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="prefixEdit"/>
          </item>
          <item row="5" column="3">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Suffix</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4" colspan="2">
           <widget class="QLineEdit" name="suffixEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <widget class="QCheckBox" name="showMinorLabelsCheck">
            <property name="text">
             <string>Show minor labels</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QCheckBox" name="showMajorLabelsCheck">
            <property name="text">
             <string>Show major labels</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QComboBox" name="notationCombo"/>
          </item>
          <item row="2" column="2">
           <widget class="GuiUtils::ColorPushButton" name="axisLabelColorButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="GuiUtils::FontPushButton" name="fontButton">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Decimal places</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="showPlusSignCheck">
            <property name="text">
             <string>Plus sign</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QCheckBox" name="showMinusSignCheck">
            <property name="text">
             <string>Minus sign</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QSpinBox" name="fieldWidthSpinBox">
            <property name="specialValueText">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Preview</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="demoLayout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuiUtils::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/colorpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GuiUtils::FontPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/fontpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>gridMajorCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>gridMinorCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gridMajorCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>gridHorizontalCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gridMajorCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>gridVerticalCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
